src/features/web-compat.js

```diff
@@ -43,6 +87,59 @@ export default class WebCompat extends ContentFeature {
         if (this.getFeatureSettingEnabled('presentation')) {
             this.presentationFix()
         }
+
+        if (this.getFeatureSettingEnabled('webShare')) {
+            this.shimWebShare()
+        }
+    }
+
+    /** Shim Web Share API in Android WebView */
+    shimWebShare () {
+        if (typeof navigator.canShare === 'function') return
```

Comment:
Should the if checks be combined?

EG:
```suggestion
        if (typeof navigator.canShare === 'function' || typeof navigator.share === 'function') return
```

Seems like it might be unexpected to just override one.

Reply:
agreed, good idea